src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -67,9 +67,7 @@
 `{command_name}`
 
 **Errors:**
-```
-{errors_string}
-```
+_Put here the error message you received. Make sure to redact sensitive information, such as personally identifiable information (PII), user name, password, credential, etc._
```

Comment:
```suggestion
_Insert here the error message you have received. Make sure to redact any sensitive information, such as user name, password, credential, subscription id, etc._
```

Reply:
I think PII is a well-known term. 🙂 https://www.dol.gov/general/ppii

Reply:
I choose the word "put" because we already use it in the following section.

https://github.com/Azure/azure-cli/blob/9cf273b8070ba4bdeb17a92bf7223ef39fce94da/src/azure-cli/azure/cli/command_modules/feedback/custom.py#L77

As for "have", do we really have to use present perfect tense? Simple past tense "received" feels good to me. No need to emphasize the "have received" status.

Also, "any" only serves as an emphasis, so it seems a little bit redundant. But, I am not a native speaker, so may be wrong.